### ISSUE TITLE
The question about “failed to compute LPC. Please try again with different parameters”

### ISSUE THREAD

**Issue by aTTracT10** [r9y9/pysptk#62]
I am very happy to find that your library can calculate LPC coefficients. Because the pyhton package named “audiolazy” is very slow to calculate. So I tried this package from "pip install" to test the accuracy. But I regret to find that many frames cannot be calculated with the message  “failed to compute ......”. While audiolazy or MATLAB can calculate them correctly. In frames that can be calculated, the first coefficient of the output is not 1 and the other coefficients are correct. Are these pysptk bugs or my wrong operations?

There is the code of calculating LPCs by different packages

```
import librosa
import numpy as np
import audiolazy.lazy_lpc as alpc
import pysptk.sptk as pplc


def readWave(filename, frame_len):
    wave_data, fs = librosa.load(filename, sr=None, mono=False, dtype='float64')
    length = len(wave_data)
    n = frame_len / 2
    last = length % n
    if last != 0:  # If samples are not an integer multiple of N/2, then zeros should be filled in.
        new_len = length + n - last
        wave_data = np.append(wave_data, np.zeros(int(n - last)))
        frame = int(new_len / n - 1)
    else:
        frame = int(length / n - 1)
    return wave_data, frame, fs


# get lpcs by audiolazy package
def getALPC(wav, frame, frame_len, order):
    n = int(frame_len / 2)
    win = np.hanning(frame_len)
    wav_frame = np.empty([frame, frame_len]) 
    lpc_frame = np.empty([frame, order + 1])
    for i in range(0, frame):
        wav_frame[i, :] = wav[i * n: (i + 2) * n] * win
        lpc_frame[i, :] = alpc.lpc(wav_frame[i, :], order).numerator
    return lpc_frame, wav_frame

# get lpcs by pysptk package
def getPLPC(wav, frame, frame_len, order):
    n = int(frame_len / 2)
    win = np.hanning(frame_len)
    wav_frame = np.empty([frame, frame_len])  
    lpc_frame = np.empty([frame, order + 1])
    for i in range(0, frame):
        wav_frame[i, :] = wav[i * n: (i + 2) * n] * win
        lpc_frame[i, :] = pplc.lpc(wav_frame[i, :], order)
    return lpc_frame, wav_frame


if __name__ == '__main__':
    frame_len = 512
    order = 8
    file_name = r"E:\Database\Hurricane_speech_corpus\mrt\lombard_mrt\mrt_001_2.wav"               
    wav_in, frame, fs = readWave(file_name, frame_len)
    lpc_frame0, wav_frame0 = getALPC(wav_in, frame, frame_len, order)
    lpc_frame1, wav_frame1 = getPLPC(wav_in, frame, frame_len, order)
```

**Comment by sehsanh** [comment]
Hello,
I have same problem , and I CHECK IT BY MATLAB  and audiolazy too. Does any one has a solution for it?!

**Comment by r9y9** [comment]
pysptk is just a wrapper of SPTK. If you have questions about SPTK itself, please ask to the team instead.

**Comment by r9y9** [comment]
Had a chance to look into this. This should be fixed by #64.
